lale/lib/snapml/snap_boosting_machine_regressor.py

```diff
@@ -11,12 +11,16 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from packaging import version
+
 try:
     import snapml  # type: ignore
 
     snapml_installed = True
+    snapml_version = version.parse(getattr(snapml, "__version__"))
```

Comment:
Now that the version is needed as well, please remove `snapml_installed`, replacing it with `snapml_version`.

anywhere that we are currently checking for `snampl_installed` would change to `snapml_version is not None`.  See https://github.com/IBM/lale/blob/master/lale/lib/snapml/snap_random_forest_classifier.py#L30-L32
for an example

Reply:
I've pushed the change for it in https://github.com/IBM/lale/pull/1309/commits/244cb17bd54a29cd141504ac6ca1456f6f33e31b